frank-flow/src/frontend/src/app/shared/models/xml-parse-error.model.ts

```diff
@@ -0,0 +1,42 @@
+export class XmlParseError {
+  startLine: number;
+  startColumn: number;
+  endLine: number;
+  endColumn: number;
+  message: string;
+
+  constructor(startLine: number, startColumn: number, message: string) {
```

Comment:
Instead of parameters use an object called settings.
https://github.com/labs42io/clean-code-typescript#functions